app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -10,17 +10,23 @@
 
     var trackingOptions = getOptionsFromCookie();
 
-    // Track initial pageview
-    this.trackPageview(null, null, trackingOptions);
-
-    // Begin error and print tracking
-    GOVUK.analyticsPlugins.error({filenameMustMatch: /gov\.uk/});
-    GOVUK.analyticsPlugins.printIntent();
-    GOVUK.analyticsPlugins.mailtoLinkTracker();
-    GOVUK.analyticsPlugins.externalLinkTracker();
-    GOVUK.analyticsPlugins.downloadLinkTracker({
-      selector: 'a[href*="/government/uploads"], a[href*="assets.publishing.service.gov.uk"]'
-    });
+    // We're setting the client ID inside a callback function because
+    // we don't have access to the client ID until GA returns a tracker object.
+    ga(function (tracker) {
```

Comment:
It's already using global.ga inside the init file: https://github.com/alphagov/static/blob/8ab43d8348dcd264302ba4c94ee16c0ba5344ee3/app/assets/javascripts/analytics/init.js.erb#L25

It's receiving global as a param in there (on line 1) so even if we removed the use of ga in the `static-analytics.js` code, we'd still be able to access it because we're receiving ga through the global param in the init file. 

What is the purpose of removing a direct link to Google Analytics?